Changelog.md

```diff
@@ -14,6 +14,7 @@ __New Features__
   - Adds detailed performance data inputs for variable-speed air source HVAC systems.
   - Add soil and moisture type arguments (for determining ground conductivity and diffusivity) and optional geothermal loop arguments for ground source heat pumps.
   - The "Geometry: Building Number of Units" input is now written to the HPXML `NumberofUnitsInBuilding` element.
+  - Updated water heater installation location defaulting to match ANSI 301-2022
```

Comment:
Need to pull this out one level; this change is not specific to the BuildResidentialHPXML measure.

```suggestion
- Updated water heater installation location defaulting to match ANSI 301-2022
```